### PR TITLE
[Review] Request from '' @ 'yast/ycp-killer/readme-usage-improvements'

### PR DESCRIPTION
**Pull request by dmajda**
Please review the following changes:
- ea62fe6 README.md: Remove examples of executed commands
- 41aa3a1 README.md: Improve the "Usage" section
- c121350 README.md: Update "yk help" output to match reality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,43 +225,41 @@ gigabytes, so make sure you have enough free space.
...
+      yk reset <module>...        # Revert module(s) work directory to clean state
+      yk restructure <module>...  # Change module(s) work directory structure to fit the Y2DIR scheme
+      yk ruby <module>...         # Convert module(s) to Ruby
+      yk submit <module>...       # Submit source files to build service for module(s)
+      yk test <module>...         # Run upstream tests for module(s)
+      yk ybc <module>...          # Compile module(s) to ybc
+
+    Options:
+      [--debug]      # verbosely log what commands are run
+      [--with-deps]  # also include module dependencies in operations
```
Comment (jreidinger) [inline README.md:258]: for me it is quite unfamiliar to add --with-deps after command and not before. So it should be explicitelly mentioned here, where options are expected

Comment (dmajda) [inline README.md:258]: I can't influence the output, it is autogenerated by Thor.

Comment (dmajda) [inline README.md:258]: (And I don't want to differ here for maintenance reasons.)

Comment (jreidinger) [inline README.md:258]: OK, then lets merge it

**Pull request merged**